ES/ESum042.xml

```diff
@@ -644,250 +355,425 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        </layoutDesc>                        
+                     </objectDesc>
+                     
+                     <handDesc>                       
+                        <handNote xml:id="h2" script="Ethiopic" corresp="#p2 #p3_i1.4 #p3_i3 #p3_i4">
+                             <locus from="126r" to="138v"/>
+                            <persName role= "scribe" ref="PRS13967WaldaTensae">Walda Tǝnśāʾe</persName>                            
+                        <seg type="script">18th-century script.</seg>
+                        <seg type="ink">Black, red.</seg>
+                             <seg type="rubrication"/>
+                             <desc>Careful, but not very regular; by a mediocre scribe. 
+                                Very similar to <ref target="#h1">Hand 1</ref> but 
+                                inferior to it. Rounded letter shapes. <persName role= "scribe" ref="PRS13967WaldaTensae">Walda Tǝnśāʾe</persName> is mentioned 
+                                as the scribe on <locus target="#138r"/>.</desc>                           
+                           </handNote>
+                     </handDesc>
+                     </physDesc>
+                     
+                     <history>
+                        <origin>
+                           <origDate notBefore="1750" notAfter="1800">Second half of the 18th century</origDate>.
+                        <ref target="#2">Unit 2</ref> is of somewhat later time than <ref target="#1">Unit 1</ref>; 
+                           it was unlikely meant to be an independent circulation unit. 
+                           The text was copied by <persName ref="PRS13967WaldaTensae">Walda Tǝnśāʾe</persName> (see <locus target="#138r"/>). 
+                           Priest <persName ref="PRS13964KeflaMaryam">Kǝfla Māryām</persName> is mentioned as the commissioner on <locus target="#138r"/>, 
+                           <persName ref="PRS13966WalattaKrestos" role="owner">Walatta Krǝstos</persName>, mentioned several times, might be his relative. 
+                        </origin>
+                     </history>
+                     </msPart>
+               
+                     <msPart xml:id="p3">
+                   <msIdentifier/>
+                   <msContents>
+                     <summary/>
+                      
+                      <msItem xml:id="p3_i1">
+                         <locus from="139" to="144v"/>
+                         <title type="complete" ref="LIT4669TaammeraQi">4 miracles of Cyricus</title>
+                         <textLang mainLang="gez"/>
+                         
+                         <msItem xml:id="p3_i1.1">
+                            <locus from="139ra" to="139vb"/>
+                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 1. About a pious king who accept to be a priest in the land of Roḥa</title>
```

Comment:
point to precise miracles, the ref to LIT4669TaammeraQi is already provided at p3_i1

Reply:
LIT5732MiracleCyriacus [Miracle of St Cyricus (Qirqos): St. Cyricus and King in the region of Rohā](https://betamasaheft.eu/LIT5732MiracleCyriacus)

Reply:
ok, thanks.